extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -228,7 +228,8 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
 
     super.open(shouldAnimate);
 
-    this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, false);
+    const pageId = this.storeService_.get(StateProperty.CURRENT_PAGE_ID);
+    this.storeService_.dispatch(Action.SET_ATTACHMENT_PAGE_ID, pageId);
```

Comment:
Let's keep the call to `Action.TOGGLE_SYSTEM_UI_IS_VISIBLE` explicit here please, it makes it easier to see who's using a given action with a simple search.